packages/cli/tests/e2e/bot/CommandBotHappyPathCommon.ts

```diff
@@ -21,10 +21,10 @@ import {
 } from "../commonUtils";
 import AppStudioLogin from "../../../src/commonlib/appStudioLogin";
 import { environmentManager } from "@microsoft/teamsfx-core";
-
+import { Runtime } from "@microsoft/teamsfx-core/src/plugins/resource/bot/v2/enum";
```

Comment:
Can we use the similar package import instead of file import? e.g. import { environmentManager } from "@microsoft/teamsfx-core"; in L23?